ledger/participant-integration-api/src/main/scala/platform/apiserver/ApiServices.scala

```diff
@@ -15,7 +15,7 @@ import com.daml.ledger.api.health.HealthChecks
 import com.daml.ledger.api.v1.command_completion_service.CompletionEndRequest
 import com.daml.ledger.client.services.commands.CommandSubmissionFlow
 import com.daml.ledger.participant.state.index.v2._
-import com.daml.ledger.participant.state.v1.WriteService
+import com.daml.ledger.participant.state.{v1 => state}
```

Comment:
How about `v1state`?

Reply:
Or simply leaving `v1.state`

Reply:
Well, that defeats the point. A later commit will replace this line with:

```scala
import com.daml.ledger.participant.state.{v2 => state} 
```

The point is that the rest of the code then doesn't need to change.

Reply:
Isn't it useful to also read which version of the state the code is dealing with (without scrolling up to the imports, that is)?

Reply:
(Even though there are really also other ways to know in an IDE, so a very minor point)

Reply:
I'm not convinced it is. I don't particularly care about the versions of anything else I use; why is this different?

Reply:
Because it's our code so it's about orientation; still, I think it's minor.

Reply:
The reference `state` should be always to the latest version of the participant state API, as we won't be supporting different versions apart from having adaptors till we deprecate the previous version.

Reply:
> apart from having adaptors till we deprecate the previous version

Alright.